packages/rum-core/src/domain/assembly.spec.ts

```diff
@@ -47,7 +57,7 @@ describe('rum assembly', () => {
           serverRumEvents.push(serverRumEvent)
         )
         startRumAssembly(
-          configuration,
+          combine(configuration, extraConfigurationOptions),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏In spec files, feel free to use spread:
```suggestion
          {...configuration, ...extraConfigurationOptions},
```